File: beginner/1048.cpp
#include <stdio.h>
 
int main() {
	float base_salary, new_salary, money_earned;
	int percentage;
	
	scanf("%f", &base_salary);
	
	if(base_salary <= 400.00) {
		new_salary = base_salary * 1.15;
		money_earned = new_salary - base_salary;
		percentage = 15;
		
	} else if (base_salary >= 400.01 && base_salary <= 800.00) {
		new_salary = base_salary * 1.12;
		money_earned = new_salary - base_salary;
		percentage = 12;
	
	} else if (base_salary >= 800.01 && base_salary <= 1200.00) {
		new_salary = base_salary * 1.10;
		money_earned = new_salary - base_salary;
		percentage = 10;
	
	} else if (base_salary >= 1200.01 && base_salary <= 2000.00) {
		new_salary = base_salary * 1.07;
		money_earned = new_salary - base_salary;
		percentage = 7;
	
	} else {
		new_salary = base_salary * 1.04;
		money_earned = new_salary - base_salary;
		percentage = 4;
	
	}
	
	printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %d %\n", new_salary, money_earned, percentage);
 
    return 0;
}

File: C#/begginer/1048.cs
using System; 

class URI {

  static void Main(string[] args) { 
    double salary = double.Parse(Console.ReadLine());
    double[] range =  { 0, 400, 800, 1200, 2000 };
    double[] percent = { 0.15, 0.12, 0.10, 0.07, 0.04 };
    double newSalary = 0;
    
    if(salary > 2000) {
      newSalary = salary * (1 + percent[4]);
      Console.WriteLine($"Novo salario: {newSalary:F2}");
      Console.WriteLine($"Reajuste ganho: {newSalary - salary:F2}");
      Console.WriteLine($"Em percentual: {percent[4] * 100:F0} %");
    } else {
      for(int i = 0; i < range.Length - 1; i++) {
        if(salary > range[i] && salary <= range[i + 1]) {
          newSalary = salary * (1 + percent[i]);
          Console.WriteLine($"Novo salario: {newSalary:F2}");
          Console.WriteLine($"Reajuste ganho: {newSalary - salary:F2}");
          Console.WriteLine($"Em percentual: {percent[i] * 100:F0} %");
          break;
        }
      }
    }
  }
}
